### PR TITLE
Added a "Request for Comments" note

### PR DESCRIPTION
**Pull request by mikehoffms**

### CODE REVIEW

File: microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md [hunk 1]
```diff
@@ -3,13 +3,16 @@ description: The Add-ons API Reference, for REST endpoints to automate publishin
...
 ms.topic: conceptual
 ms.prod: microsoft-edge
 keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, add-ons api, publish api
 ---
 # Microsoft Edge Add-ons API Reference
 
+> [!NOTE]
+> This article is a Request for Comments.  The Microsoft Edge Add-ons API is not yet available for testing, and the Publish APIs page is not yet available at Partner Center.  The Microsoft Edge Add-ons API is under active development and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here are not exhaustive and are subject to change.
+
 This is the REST endpoint reference for the Microsoft Edge Add-ons API.  This API automates publishing updates to add-ons that have been submitted to the Microsoft Edge Add-ons website.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md:16]: Add here this paragraph from top of other article, w/ link to enter an Issue?
"This article, along with the Microsoft Edge Add-ons API Reference, provides an overview of the proposed Microsoft Edge Add-ons API. We look forward to your suggestions and feedback on the proposed API contracts. Please submit your feedback as an Issue about the Add-ons API."

File: microsoft-edge/extensions-chromium/publish/api/using-addons-api.md [hunk 2]
```diff
@@ -3,19 +3,20 @@ description: REST endpoints to automate publishing updates to add-ons that are s
...
-ms.date: 08/10/2021
+ms.date: 08/19/2021
 ms.topic: conceptual
 ms.prod: microsoft-edge
 keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, add-ons api, publish api
 ---
 # Using the Microsoft Edge Add-ons API
 
+> [!NOTE]
+> This article is a Request for Comments.  The Microsoft Edge Add-ons API is not yet available for testing, and the Publish APIs page is not yet available at Partner Center.  The Microsoft Edge Add-ons API is under active development and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here are not exhaustive and are subject to change.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/api/using-addons-api.md:14]: merge the feedback link into the Note?  (on both pages)
Merge these two, temporary intro/setup paragraphs?

File: microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md (hunk 1 above)
Comment (Reezaali) [reply microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md:16]: I think adding that would be a good idea.

File: microsoft-edge/extensions-chromium/publish/api/using-addons-api.md [hunk 3]
```diff
@@ -3,19 +3,20 @@ description: REST endpoints to automate publishing updates to add-ons that are s
 title: Using the Microsoft Edge Add-ons API
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 08/10/2021
+ms.date: 08/19/2021
 ms.topic: conceptual
 ms.prod: microsoft-edge
 keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, add-ons api, publish api
 ---
 # Using the Microsoft Edge Add-ons API
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/publish/api/using-addons-api.md:11]:
```suggestion
# Using the Microsoft Edge Add-ons API (under development)
```

File: microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md [hunk 4]
```diff
@@ -3,13 +3,16 @@ description: The Add-ons API Reference, for REST endpoints to automate publishin
 title: Microsoft Edge Add-ons API Reference
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 08/10/2021
+ms.date: 08/19/2021
 ms.topic: conceptual
 ms.prod: microsoft-edge
 keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, add-ons api, publish api
 ---
 # Microsoft Edge Add-ons API Reference
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md:11]:
```suggestion
# Microsoft Edge Add-ons API Reference (under development)
```

**Review by Reezaali — CHANGES_REQUESTED** [review]

**Review by Reezaali — APPROVED** [review]

**Pull request merged**